### PR TITLE
Bitmark References: Node Setup Edit

### PR DESCRIPTION
**Pull request by shannona**
Some general setup for the Bitmark References & full edit of Bitmark Node Seutp.

### CODE REVIEW

**Comment by vienlt** [comment]
@hxw: Shannon edited the node setup document which the content is mainly taken from your bitmarkd readme file. Could you help to review the edited version? Thanks.

**Review by hxw — APPROVED** [review]

File: pages/bitmark-references/node-setup/bitmark-node-setup.md [hunk 1]
```diff
@@ -1,54 +1,35 @@
...
+#  Bitmark Node Installation
 
-A full-node of bitmarkd consists of a bitmard, a recorderd, a payment system, a litecoind, and a bitcoind services. For a beginner, it is easy and fast to run a bitmark-node-docker which wrapped all components and settings. For a developer or someone who wants to study the main program, the person can run bitmarkd or recorderd service directly. 
-Bitmarkd project is the main program of bitmark node. It consists bitmarkd service for verifying and recording transactions in the Bitmark blockchain and recorderd service for computing the Bitmark proof-of-work algorithm that allows nodes to compete to win blocks on the Bitmark blockchain. 
+The [Bitmark Quickstart](../../learning-bitmark/quick-start) includes a tutorial on [installing the Bitmark Property System using docker](../../learning-bitmark/quick-start/simple-solution-for-node-setup.md). That setup is intended for beginners who want to make it fast and fast and easy to run the Bitmark Property System using a `docker` container that wraps all components and settings. Developers, or people who want to study the main program, may instead choose to run `bitmarkd` or `recorderd` services directly.
```
Comment (vienlt) [inline pages/bitmark-references/node-setup/bitmark-node-setup.md:13]: > want to make it fast and fast and easy
Is the "fast and fast" intentional?

**Review by vienlt — APPROVED** [review]
It looks good to me, only one comment about the typos.

File: pages/bitmark-references/node-setup/bitmark-node-setup.md (hunk 1 above)
Comment (shannona) [reply pages/bitmark-references/node-setup/bitmark-node-setup.md:13]: Thanks, fixed.

**Comment by shannona** [comment]
I'm still not showing authorization to merge, but it looks like this one is ready.

**Pull request merged**